shipyard-controller/models/project.go

```diff
@@ -117,7 +118,32 @@ func (updateProjectParams *UpdateProjectParams) Validate() error {
 		return errors.New("project name missing")
 	}
 	if !keptncommon.ValidateKeptnEntityName(*updateProjectParams.Name) {
-		return errors.New("provided project name is not a valid Keptn entity name")
+		errorMsg := "Project name contains upper case letter(s) or special character(s).\n"
```

Comment:
I would not tell what is wrong but rather only say what we accept. This way we need to maintain only one place 😉 
Furthermore, error messages should start with a lower case and don't be verbose. These details should be covered by documentation and not by error messages.